core/arch/arm/tee/cache.c

```diff
@@ -49,13 +49,15 @@ TEE_Result cache_operation(enum utee_cache_operation op, void *va, size_t len)
 
 	switch (op) {
 	case TEE_CACHEFLUSH:
+#ifdef CFG_PL310
```

Comment:
I think this is correct, but I am concerned with the terminology. I mean, `cache_maintenance_l2()` does nothing when `CFG_PL310` is not set, yet the platform is very likely to have a L2 cache. I find it confusing. I think we should introduce "outer" in the function name. How about `cache_maintenance_outer_l2()`?